### PR TITLE
Added 'FORCE_WORLD_COPY' environment variable

### PR DESCRIPTION
**Pull request by Trikolon**
Setting `FORCE_WORLD_COPY=TRUE` will overwrite the world on server start.

### CODE REVIEW

**Comment by Trikolon** [comment]
Wasn't sure where to mention this in the README. Feel free to move it  / let me know if you find a better fit.

File: start-finalSetupWorld [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ "$WORLD" ]] && [ ! -d "$worldDest" ]; then
     mv "$baseDir" "$worldDest"
```
Comment (itzg) [inline start-finalSetupWorld:40]: This will need to switch to rsync also.

Comment (Oekn5w) [inline start-finalSetupWorld:40]: I think that line fails, if the directory already exists

**Comment by Oekn5w** [comment]
I think we should store the world that is to be synced / copied, in the case of a zip or download, we'd have to decompress / download every time, possibly overwriting the `WORLD` variable as long as the container lives

**Comment by Trikolon** [comment]
> I think we should store the world that is to be synced / copied, in the case of a zip or download, we'd have to decompress / download every time, possibly overwriting the `WORLD` variable as long as the container lives

The tricky part is how to invalidate this cache. I agree that this could be optimized depending on the use case. But I think we can do that as follow up work if it's actually needed.

File: README.md [hunk 2]
```diff
@@ -947,6 +947,9 @@ read-only volume attachment to ensure the clone source remains pristine.
 docker run ... -v $HOME/worlds:/worlds:ro -e WORLD=/worlds/basic
 ```
 
+### Overwrite world on start
```
Comment (itzg) [inline README.md:950]: Yep, this was a perfect spot for it.

**Comment by itzg** [comment]
`latest` image is now pushed. I'll cut a new release, 1.5.0, and merge into other branches/tags.

**Comment by itzg** [comment]
and released/merged https://github.com/itzg/docker-minecraft-server/releases/tag/1.5.0

**Pull request merged**